kortex_api/CMakeLists.txt

```diff
@@ -20,7 +23,11 @@ project(kortex_api)
 
 #Current: only support Linux x86_64
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+  FetchContent_MakeAvailable(kinova_binary_api)
   set(API_URL https://artifactory.kinovaapps.com:443/artifactory/generic-public/kortex/API/2.5.0/linux_x86-64_x86_gcc.zip)
```

Comment:
Would it make sense to bump the x86_64 version up to 2.6.0 as well?

Reply:
What version of firmware do we have on the arm? It does appear [Kinova recommends](https://github.com/Kinovarobotics/kortex?tab=readme-ov-file#download-links) 2.6.0 for the gen3

Reply:
The Kinova folks told us to use 2.6.0 version so the Arm processor, so I imagine that it would be the same version for the x86 version.

Reply:
It would be good to double check the firmware we are running on the arms to make sure they are up to date.